Program/Seasons.py

```diff
@@ -0,0 +1,222 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
+setMediaPath("F:\COMP 120 Tinkering Graphics-JH182233\Pictures") 
+picfour = makePicture("Tree-Four-Seasons.jpg") 
+picwinter = makePicture("Tree-Winter.jpg") 
+picspring = makePicture("Tree-Spring.jpg") 
+picsummer = makePicture("Tree-Summer.jpg")
+picautumn = makePicture("Tree-Autumn.jpg")  
+picM = makeEmptyPicture(513, 556, black)
+pic = makeEmptyPicture(1025, 1110, black)
+
+
+#The main function for the program
+def main():
+    copy()
+    request()
+    request2()
```

Comment:
Avoid using numbers at the end of function or variable names. There is no way to distinguish between these two functions. 

Either either define two different functions with each having a more meaningful name, or use an argument to produce different behavior within the same function.